Functions.Templates/Templates/SqlInputBinding-PowerShell/function.json

```diff
@@ -0,0 +1,27 @@
+{
+    "bindings": [
+      {
+        "authLevel": "function",
+        "type": "httpTrigger",
+        "direction": "in",
+        "name": "Request",
+        "methods": [
+          "get",
+          "post"
```

Comment:
This doesn't really make sense to have as POST - it's just returning data. (and the other input binding templates are get only as well)